### PR TITLE
SLA-1929 Create loading screen component

### PR DESCRIPTION
**Pull request by donbernar**
## Refs

- **Issue**: Resolves [SLA-1929](https://linear.app/slashauth/issue/SLA-1929/sr-singin-component-create-loading-screen)

## What?

Create the loading screen component

## Why?

Its part of the v1 redesign

## Screenshots:

image (9)

### CODE REVIEW

File: src/core/ui/components/sign-in/screens/loading.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            {selectedLoginMethod ? `to ${selectedLoginMethod?.name}` : ''}
+          </Text>
+          <Text
+            className={classNames(
+              text.centered,
+              margin.top2,
+              styles.detailedDescription
+            )}
+          >
+            Please sign the message in your wallet app or plugin to continue
```
Comment (nedrocks) [inline src/core/ui/components/sign-in/screens/loading.tsx:52]: Given this is only for wallets do we want to have a different set of options or a different component for non-wallet logins?

**Review by nedrocks — APPROVED** [review]

File: src/core/ui/components/sign-in/screens/loading.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { BaseButton } from '../../primitives/button';
+import { classNames } from '../../../../../shared/utils/classnames';
+import text from '../../primitives/text.module.css';
+import margin from '../../primitives/margin.module.css';
+import styles from './loading.module.css';
+
+export const LoadingScreen = ({ navigateBack }) => {
+  const { selectedLoginMethod: selected } = useLoginMethods();
+
+  const selectedLoginMethod = selected;
```
Comment (donbernar) [inline src/core/ui/components/sign-in/screens/loading.tsx]: Gotta remove this

File: src/core/ui/components/sign-in/screens/loading.tsx (hunk 1 above)
Comment (donbernar) [reply src/core/ui/components/sign-in/screens/loading.tsx:52]: What would the texts look like for non-wallet logins?

File: src/core/ui/components/sign-in/screens/loading.tsx (hunk 2 above)
Comment (nedrocks) [reply src/core/ui/components/sign-in/screens/loading.tsx]: is this still a todo for this PR?

File: src/core/ui/components/sign-in/screens/loading.tsx (hunk 1 above)
Comment (nedrocks) [reply src/core/ui/components/sign-in/screens/loading.tsx:52]: We discussed this morning -- we should update this but if it makes sense to do it in a future PR that's fine too!

**Review by nedrocks — APPROVED** [review]

File: src/core/ui/components/sign-in/screens/loading.tsx (hunk 2 above)
Comment (donbernar) [reply src/core/ui/components/sign-in/screens/loading.tsx]: Sorry, I've just pushed the fix! I will merge this PR shortly

**Pull request merged**